cmd/aro/lb.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	mgmtfeatures "github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2021-07-01/features"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/davecgh/go-spew/spew"
+
+	"github.com/sirupsen/logrus"
+)
+
+func lb(ctx context.Context, log *logrus.Entry) error {
+
+	authorizer, err := auth.NewAuthorizerFromCLIWithResource("https://management.azure.com/")
```

Comment:
Should `https://management.azure.com/` be cloud-aware? Do we anticipate this problem existing in FF?